test/libsolidity/syntaxTests/constants/define_constants_with_external_constant_member_var.sol

```diff
@@ -0,0 +1,11 @@
+uint256 constant MAX = 1;
+
+library Internal {
+    uint256 internal constant SCALE = 100;
+}
+
+contract Usage {
+    uint256 private constant LIMIT = MAX * Internal.SCALE;
+}
```

Comment:
Can you add a case with a deeper tree of dependencies that goes through several levels? We should especially have one that ensures that `LIMIT` can now be used as a value in another constant definition.

Reply:
Done